python/create_service_diagnostics_bundle.sh

```diff
@@ -5,7 +5,7 @@ set -eu -o pipefail
 readonly SCRIPT_DIRECTORY="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
 readonly DCOS_SERVICE_DIAGNOSTICS_SCRIPT_PATH="dcos-sdk-service-diagnostics/python"
 
-readonly VERSION='v0.4.0'
+readonly VERSION="$(head -1 < "$SCRIPT_DIRECTORY/VERSION")"
```

Comment:
This would also work :-)

```suggestion
readonly VERSION="$(head -1 "$SCRIPT_DIRECTORY/VERSION")"
```

Reply:
changed